packages/fluentui/react/src/components/Button/Button.tsx

```diff
@@ -193,6 +193,10 @@ const Button: React.FC<WithAsProp<ButtonProps>> &
   };
 
   const handleFocus = (e: React.SyntheticEvent) => {
+    if (disabled) {
```

Comment:
⚠️ Can you please explain why? Does it work in the same way for plain `button`?

Reply:
Just copy pasted how it's done for handleClick.

Reply:
I suggest to remove it as by it's nature `disabled` `button` should not be focusable: https://codesandbox.io/s/focused-feather-2koj0

Reply:
ok, sure :)